README.md

```diff
@@ -92,6 +94,23 @@ else fails, try removing the virtual environment and reinstalling:
 [local5000]:http://localhost:5000/
 
 
+### Troubleshooting Possible Errors
```

Comment:
Changing this to bullet points would be much better imo. Finding the right troubleshooting steps in prose does not offer a good experience.

Reply:
Noted